src/details/ArborX_DetailsNode.hpp

```diff
@@ -23,7 +23,7 @@ namespace ArborX
 {
 namespace Details
 {
-struct Node
+struct alignas(64) Node
```

Comment:
Provisory change.  My plan is to drop in the next PR along with `ptrdiff_t -> int` for storing children.

Reply:
> I would like to consider whether that should be a configurable parameter.

What does "that" refer to?

Reply:
Whether we are storing children as size_t/ptrdiff_t, or int.

Reply:
I'd say we can fix later.  One may ask to use double precision as well ;)

Reply:
There is a lot more involved for making things work with more than 2B objects.
For instance we explicitly take indices as `int` in our lambdas/functions.  Also default index types would need to change in execution policies.